specification/dynamicstelemetry/resource-manager/Microsoft.DynamicsTelemetry/preview/2019-01-24/dynamicstelemetry.json

```diff
@@ -0,0 +1,135 @@
+{
+  "swagger": "2.0",
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "info": {
+    "title": "Dynamics Telemetry",
+    "version": "2019-01-24"
+  },
+  "host": "telemetry.dynamics.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.DynamicsTelemetry/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": true,
+            "description": "",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Operation"
+              }
```

Comment:
Could you extract this definition and move to definition section?, see this [example](https://github.com/Azure/azure-rest-api-specs/blob/32b0d873aa851d456dfde7d6ba1d89ff33f897f0/specification/addons/resource-manager/Microsoft.Addons/preview/2017-05-15/Addons.json#L49)

Reply:
Question: double checking, do you support pagination for operations API?

Reply:
> Could you extract this definition and move to definition section?, see this [example](https://github.com/Azure/azure-rest-api-specs/blob/32b0d873aa851d456dfde7d6ba1d89ff33f897f0/specification/addons/resource-manager/Microsoft.Addons/preview/2017-05-15/Addons.json#L49)

Done

Reply:
> Question: double checking, do you support pagination for operations API?

No